src/AppBundle/PullRequests/BodyParser.php

```diff
@@ -79,6 +81,14 @@ public function getCategory()
         return $this->extractWithRegex($regex);
     }
 
+    /**
+     * @return bool
+     */
+    public function isTestCategory()
+    {
+        return 1 === preg_match('/\\bTE\\b/', $this->getCategory());
```

Comment:
```suggestion
        return 1 === preg_match('/\bTE\b/', $this->getCategory());
```
You don't need to escape backslash in single quote context